### DISCUSSION TITLE
支持mac吗？

### DISCUSSION THREAD

**Discussion by river1987** [labring/sealos#3144] [General]
按照官网安装时报错，是不是现在还不支持MAC OS？

curl -sfL  https://raw.githubusercontent.com/labring/sealos/v4.2.1-rc3/scripts/install.sh \                                                              
    | sh -s v4.2.1-rc3 labring/sealos
[INFO]  Using v4.2.1-rc3 as release
[INFO]  Using labring/sealos as your repo
[INFO]  Downloading tar curl https://github.com/labring/sealos/releases/download/v4.2.1-rc3/sealos_4.2.1-rc3_linux_amd64.tar.gz
[INFO]  Downloading sealos, waiting...
x sealos
[INFO]  Installing sealos to /usr/bin/sealos
Password:
chown: river: illegal group name
[WARN]  Failed to install sealos

**Comment by cuisongliu** [comment]
Mac无法支持，只支持Linux
